lib/mortar/command.rb

```diff
@@ -153,6 +151,8 @@ def build_kubeconfig_from_env
         preferences: {},
         current_context: 'mortar'
       )
+    rescue ArgumentError
+      signal_usage_error "KUBE_TOKEN env doesn't seem to be valid base64 encoding!"
```

Comment:
"doesn't seem to be valid base64 encoding" still sounds wrong.

Reply:
changed to `KUBE_TOKEN env doesn't seem to be base64 encoded!`, ok now?

Reply:
yes :)